src/Microsoft.AspNetCore.Razor.Runtime/TagHelpers/TagHelperContent.cs

```diff
@@ -167,5 +173,17 @@ IHtmlContentBuilder IHtmlContentBuilder.Clear()
         {
             return Clear();
         }
+
+        /// <inheritdoc />
+        void IHtmlContentContainer.MoveTo(IHtmlContentBuilder destination)
+        {
+            if (destination == null)
```

Comment:
Good point. `null` checks are missing in `CopyTo()` as well as aspnet/HttpAbstractions#11 in general.